### ISSUE TITLE
Bypass LAN and private networks

### ISSUE THREAD

**Issue by shanghaix** [Jigsaw-Code/outline-client#355]
Hello,

Using the latest update on Mac and iOS, the release notes state that now Outline Client bypasses LAN and private networks while connected. 
Unfortunately I used outline to access a lot of private resources on remote networks (SSH...) and with the new update I am unable to do it anymore.
Could it be possible to let the user chose the default behaviour instead of pushing the feature to everyone?

**Comment by trevj** [comment]
@shanghaix Thanks for using Outline and for reporting this. I'm afraid that in addition to this client change, we also recently removed support for accessing private networks in the *server* - so even if we added this as an option in the client, it still wouldn't work (with Outline servers, at least).

Sorry this has bitten you - @sandrigo FYI and @shanghaix perhaps you could describe in a little more detail how you access those servers?

**Comment by rmonst3r** [comment]
@trevj I too use outline to access private networks, is this feature now discontinued permanently?

**Comment by trevj** [comment]
@rmonst3r Sorry for the slow reply - I can't say permanently because we could make it a feature in the client and if this issue received a lot of upvotes we would re-prioritise but for now we have no plans to change this, sorry.

**Comment by xumx** [comment]
Upvote for this feature.

I thought it was an issue with the shadowbox Docker container network and spent the whole afternoon trying to figure out what is going on. 


The usage scenario is pretty clear. An easy way to use outline (from home or while traveling) to access remote private LAN (in office) for work.

**Comment by tbublik** [comment]
It is valuable to privately access home machines connected through Outline while not at home (e.g. from a smartphone or laptop).

**Comment by neelg1193** [comment]
I also want this feature. Is it feasible to add as an option during setup?

**Comment by wiredbrother** [comment]
I agree. The reason I wanted Outline in the first place was I thought this feature was built in. Now back to the drawing board for a VPN to access my servers. Sad face indeed

**Comment by rmonst3r** [comment]
Guys try out Softether, that's what I switched to.
